product_stock_state/views/product_template_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_product_template_form" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="stock.view_template_property_form" />
+        <field name="arch" type="xml">
+            <field name="route_ids" position="before">
```

Comment:
perhaps you could put them before the group instead of the field, right now it looks a bit weird 
![image](https://user-images.githubusercontent.com/103428640/206928935-491c26d3-ab25-4513-863c-cd71b5814017.png)

Reply:
Moved inside Inventory/Logistics group